exercises/practice/perfect-numbers/.meta/config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "authors": [
+    "sbromberger"
+  ],
+  "contributors": [
```

Comment:
I'm pretty sure it is only you as an author, and you mention the version you forked from to hint that others has worked on the exercise as well.
https://exercism.org/docs/building/tracks/practice-exercises:
>- contributors: The GitHub username(s) of the exercise approach introduction's contributor(s) (optional)
>-  Including reviewers if their reviews are meaningful/actionable/actioned.

Reply:
Where do I put the credit for the version I started with (golang)?

Reply:
I mixed up concept and practice exercises. I think it should be possible in both.

So I would suggest to keep "ferhatelmas" as a contributor, but drop the rest.